public/locales/ru/dnssec.json

```
@@ -0,0 +1,98 @@
+{
+  "title": "Импортировать {{name}} в ENS",
+  "error": {
+    "unknown": "Произошла неизвестная ошибка",
+    "noTxtRecord": "Запись не найдена",
+    "dnssecFailure": "DNSSEC не включен",
+    "invalidTxtRecord": "Найдена неверная запись",
+    "invalidAddressChecksum": "Адрес недействителен",
+    "resolutionFailure": "Предоставленная запись не разрешилась"
+  },
+  "status": {
+    "checking": "Проверка...",
+    "error": "Произошла ошибка при проверке",
+    "secondsAgo": "Последняя проверка была {{count}} секунд(у) назад",
+    "aWhileAgo": "Последняя проверка была некоторое время назад",
+    "minutesAgo_one": "Последняя проверка была {{count}} минуту назад",
+    "minutesAgo_other": "Последняя проверка была {{count}} минуты назад"
+  },
+  "steps": {
+    "selectType": {
+      "title": "Претендовать на {{name}}",
+      "subtitle": "Импорт доменных имен позволяет использовать их в качестве ENS имен.",
+      "learnMore": "Узнайте больше о импорте имен",
+      "select": {
+        "heading": "Как вы хотите импортировать ваш домен?",
+        "offchain": {
+          "name": "Вне сети",
```

```suggestion
          "name": "Оффчейн",
```

"Вне сети" на английском было бы "Offline". Как вариант, можно ещё сказать "Вне блокчейна"